src/Draco.Compiler/Api/Compilation.cs

```diff
@@ -204,14 +272,48 @@ public sealed class Compilation : IBinderProvider
             Diagnostics: ImmutableArray<Diagnostic>.Empty);
     }
 
-    internal Binder GetBinder(SyntaxNode syntax) => this.binderCache.GetBinder(syntax);
+    internal ModuleSymbol GetCompilationUnitModule(SyntaxTree tree)
```

Comment:
The method makes sense, but let's rename it to `GetModuleForSyntaxTree` (if I understand what it does)